bigquery/tests/unit/test_client.py

```diff
@@ -5230,7 +5230,7 @@ def test_load_table_from_dataframe(self):
         from google.cloud.bigquery import job
 
         client = self._make_client()
-        records = [{"name": "Monty", "age": 100}, {"name": "Python", "age": 60}]
+        records = [{"id": 1, "age": 100}, {"id": 2, "age": 60}]
```

Comment:
Any special reason for replacing a string column with an integer one?
(not that there's anything wrong with it, it looks fine, just being curious)

Reply:
Since string columns have a pandas dtype of `object`, we end up hitting the soon-to-be-deprecated "no schema available" path. Alternatively, I could have used a partial schema in these tests, but that didn't seem relevant to the purpose of these tests.

Reply:
Makes sense, thanks.